docs/docs/concepts.mdx

```diff
@@ -29,24 +29,27 @@ All chains, agents, and retrieval strategies here are NOT specific to any one in
 ### `langchain-community`
 
 This package contains third party integrations that are maintained by the LangChain community.
-Key partner packages are separated out (see below).
+Key integration are separated out into their own packages (see **Integration packages** above).
 This contains all integrations for various components (LLMs, vector stores, retrievers).
 All dependencies in this package are optional to keep the package as lightweight as possible.
 
 ### [`langgraph`](https://langchain-ai.github.io/langgraph)
 
 `langgraph` is an extension of `langchain` aimed at
 building robust and stateful multi-actor applications with LLMs by modeling steps as edges and nodes in a graph.
+`langgraph` exposes high level interfaces for creating common types of agents, as well as a low-level API for composing custom flows.
 
-LangGraph exposes high level interfaces for creating common types of agents, as well as a low-level API for composing custom flows.
+Head to the [`langgraph` docs here](https://langchain-ai.github.io/langgraph).
 
 ### [`langserve`](/docs/langserve)
 
 A package to deploy LangChain chains as REST APIs. Makes it easy to get a production ready API up and running.
 
 ### [LangSmith](https://docs.smith.langchain.com)
 
-A developer platform that lets you debug, test, evaluate, and monitor LLM applications.
+A developer platform that lets you debug, evaluate, monitor, and optimize LLM applications, whether they're built with LangChain libraries or not.
```

Comment:
we've been going with "debug, test, collaborate, and monitor" in the customer decks